sys/include/net/ieee802154/radio.h

```diff
@@ -70,22 +70,42 @@ typedef enum {
     /**
      * @brief the device supports frame retransmissions with CSMA-CA
      *
-     * @note it's implicit that a radio supports @ref IEEE802154_CAP_AUTO_CSMA
-     * if this cap is available
+     * The device supports sending with CSMA-CA and retransmissions.  If the
+     * CSMA-CA fails, the device reports a @ref TX_STATUS_MEDIUM_BUSY when
+     * calling @ref ieee802154_radio_ops::confirm_transmit. In case CSMA-CA
+     * succeeds and the ACK frame is expected, the
+     * device reports a @ref TX_STATUS_SUCCESS if the ACK frame is received
+     * during any retransmission attempt. Otherwise, it reports a @ref
+     * TX_STATUS_NO_ACK
```

Comment:
Will the device do a retransmission in that case?

Reply:
nope, only if IEEE802154_CAP_FRAME_RETRANS is available.

Reply:
Better add a short note :)

Reply:
what I described there, IEEE802154_CAP_FRAME_RETRANS implies IEEE802154_CAP_AUTO_CSMA (but not the other way around).

Frame retransmissions without CSMA-CA makes no sense and it's not even supported by most platforms.

Reply:
Wait this *is* the documentation for `IEEE802154_CAP_FRAME_RETRANS`

Reply:
Ah ok reading it again it's more clear.
I missed the sentence about *retransmission attempts*.

Reply:
The thing is, AUTO_CSMA just tells the upper layer if CSMA-CA failed or if the frame was sent. The upper layer should retransmit.

Should I still add a short note?

Reply:
ah ok! great :)